### PR TITLE
Project Page css update

### LINKED ISSUE #830: Project Page mobile view needs some help.
### Overview 
We are in need of a css wizard to fix a few problems on the mobile view of the projects section on the main page
Screen Shot 2020-12-06 at 12 08 32 AM

### Details
- [x] `.project-header` and `.project-pitch` need space between them.
- [x] The styling on the filter list needs to be reimagined so that the text does not overlap the label arrow(down carrot thingy)

### PR DESCRIPTION
**Pull request by ItsLaszlo**
Fixes #830 
 
Updated CSS of dropdown and project header. Increased the right margin of .project-header to add spacing between it and .project-pitch. Added padding to the right of the filter item so the text does not overlap the label arrow.

### CODE REVIEW

**Comment by ItsLaszlo** [comment]
Screen Shot 2021-01-14 at 6 48 08 PM

**Review by trtincher — APPROVED** [review]
Looks great!

**Comment by jbubar** [comment]
This looks, Good. Hoping to talk about it in the meeting. Thanks for your patience

File: _sass/components/_project-filter.scss [hunk 1]
```diff
@@ -21,6 +21,7 @@ li.filter-item {
   background: white;
   display: flex;
   padding: 10px 15px;
+  padding-right: 30px;
```
Comment (jbubar) [inline _sass/components/_project-filter.scss]: do a margin-right: auto or something here instead

Comment (jbubar) [reply _sass/components/_project-filter.scss]: https://www.hackforla.org/

Screen Shot 2021-01-17 at 11 31 10 AM

**Review by jbubar — CHANGES_REQUESTED** [review]

File: _sass/components/_project-filter.scss (hunk 1 above)
Comment (ItsLaszlo) [reply _sass/components/_project-filter.scss]: The spacing has been corrected. It was actually within the _project.scss file that required editing to align the project pitch to the right.

**Comment by ItsLaszlo** [comment]
Screen Shot 2021-01-18 at 9 57 37 AM

**Review by trtincher — APPROVED** [review]
Looks good!

**Comment by jbubar** [comment]
Screen Shot 2021-01-20 at 2 17 46 PM
Sorry to be nitpicky, but we have run into another error, where at a certain screen width, the dropdowns extend off the screen

**Comment by ItsLaszlo** [comment]
> Screen Shot 2021-01-20 at 2 17 46 PM
> Sorry to be nitpicky, but we have run into another error, where at a certain screen width, the dropdowns extend off the screen

Problem fixed! Instead of adding padding increased the screen max-width for the class ul.filter-list which creates 2 rows and 2 columns.
image

**Comment by jbubar** [comment]
Accidentally pulled changes into this branch from danielles pr

**Comment by jbubar** [comment]
Cannot merge this until this is fixed

**Comment by ItsLaszlo** [comment]
Danielle's changes have been removed. Ready to merge!

**Pull request merged**